web/packages/teleterm/src/ui/StatusBar/useActiveDocumentClusterBreadcrumbs.ts

```diff
@@ -62,8 +62,10 @@ export function useActiveDocumentClusterBreadcrumbs(): Breadcrumb[] {
   }
 
   return [
-    { name: rootCluster.name },
-    clusterUri !== rootClusterUri && { name: cluster.name },
+    { name: `${routing.parseClusterName(rootClusterUri)}` },
```

Comment:
This is a bit tricky because if the root cluster name differs from the proxy host, then the status bar is going to say one thing and the tab title is going to say another.

But as you mention, when approaching the profile name as the identifier of the workspace, this change makes sense. And tbh I don't have a better idea. I think the guidance you laid out with a workspace vs resources is very reasonable. E.g., it makes sense that the connection tracker shows the cluster name and not the profile name.

<img width="2016" height="1388" alt="profile name in status bar" src="https://github.com/user-attachments/assets/a77da895-8c00-4b99-bfb8-37d0f03262e1" />

Reply:
Putting the cluster name in parentheses is not a bad solution tbh, we can keep it in mind in case users complain about this change.